### PR TITLE
Gh enterprise 1706

### PR DESCRIPTION
**Pull request by asabine**

### CODE REVIEW

**Comment by danantonioreyes** [comment]
tagging in maggie in this conversation @PCF-Testing 

I think having the background colors cream-snow-cream for the three sections is a little bit confusing.
The "Languages" section having it's own background color together with the submit button seems like you only have the Languages section that's actually part of the form, and there is a disconnect with the "About You" section.

My suggestion is to split the whole page into just two sections:

1. the team's text for the application page
2. the actual form which includes the "About You" and "Languages" part of the form

Like so: (zoomed-out image)
image

Another thing worth pointing out as Maggie has mentioned to me is that apparently users have trouble understanding that they need to scroll down and to respond to the questions posed in the About You and Languages section, so maybe we can add a help text somewhere like "Please fill out 'About You' and 'Languages' sections below and click 'Submit' to apply"

This help text can either be put just below the "Apply to {{team}}" header, or at the beginning of the form section as indicated in my screenshot above (I just put the text in a hurry for demonstration's sake in the screenshot)

File: templates/future/teams/applications/join.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                  {% blocktrans %}
+                  We request that each new applicant does at least one full set of subtitles within the first week of joining. Aside from that, it's up to you!
+                  {% endblocktrans %}
+                </p>
+                {% endif %}
+              </div>
+            </header>
+          </div>
+        </div>
+        {% include "future/teams/applications/forms/apply.html" %}
```
Comment (danantonioreyes) [inline templates/future/teams/applications/join.html]: You can take the form itself out and put it into a separate `<section>` in itself so you could take advantage of the styling of `.pageSection` class

File: templates/future/teams/applications/join.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{% extends "future/teams/base.html" %}
+{% load i18n forms %}
+
+{% block content %}
+<section>
```
Comment (danantonioreyes) [inline templates/future/teams/applications/join.html]: You can do something like
```
<main id="dashboard" class="pageContent">
<section class="pageSection">
```
here to imitate how the background of each section is styled for collab team dashboards

`amara-enterprise/collab/templates/collab/dashboard.html` is a good example on how to use the styling of `.pageSection`

File: templates/future/teams/applications/join.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                {% endif %}
+              </div>
+            </header>
+          </div>
+        </div>
+        {% include "future/teams/applications/forms/apply.html" %}
+      </div>
+    </div>
+  </div>
+</section>
```
Comment (danantonioreyes) [inline templates/future/teams/applications/join.html]: Then put the form in a section of its own:
```
<section class="pageSection">
{% include "future/teams/applications/forms/apply.html" %}
</section
```

**Comment by asabine** [comment]
updated the styles, the page looks like this currently. @PCF-Testing does this look acceptable?
image

**Pull request merged**